File: ecommerc_dotnet/dto/Response/AddressResponseDto.cs
using ecommerc_dotnet.module;
using NetTopologySuite.Geometries;

namespace ecommerc_dotnet.dto.Response;

public class AddressResponseDto
{ 
    public Guid id { get; set; }
    public decimal longitude { get; set; }
    public decimal latitude { get; set; }
    public string title { get; set; } 
    public bool isCurrent { get; set; } = false;
}

File: ecommerce-delivery-man/app/src/main/java/com/example/eccomerce_app/dto/response/AddressResponseDto.kt
package com.example.e_commerc_delivery_man.dto.response

import android.location.Location
import com.example.hotel_mobile.services.kSerializeChanger.UUIDKserialize
import kotlinx.serialization.Serializable
import java.util.UUID

@Serializable
data class AddressResponseDto(
    @Serializable(with = UUIDKserialize::class)
    val id: UUID,
    val longitude: Double,
    val latitude: Double,
    val title: String?,
    val isCurrent: Boolean?=false
)
